packages/botonic-react/src/webchat/webchat.jsx

```diff
@@ -735,13 +764,22 @@ export const Webchat = forwardRef((props, ref) => {
               <ErrorMessage>{webchatState.error.message}</ErrorMessage>
             </ErrorMessageContainer>
           )}
-
-          <WebchatMessageList />
-
-          {webchatState.replies &&
-            Object.keys(webchatState.replies).length > 0 && (
-              <WebchatReplies replies={webchatState.replies} />
-            )}
+          <div
+            id={BotonicContainerId.ChatArea}
+            style={{
+              display: 'inherit',
+              flexDirection: 'inherit',
+              height: chatAreaHeight,
+              width: 'inherit',
+              overflow: 'inherit',
+            }}
+          >
+            <WebchatMessageList />
+            {webchatState.replies &&
+              Object.keys(webchatState.replies).length > 0 && (
+                <WebchatReplies replies={webchatState.replies} />
+              )}
```

Comment:
This should be a component within the WebchatMessageList

Reply:
leave it for next refactors 👌